src/generateRandomNumber.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const generateRandomNumber = (min, max) => {
+  return Math.floor(Math.random() * (max - min) + min);
+};
```

Comment:
![Capture](https://user-images.githubusercontent.com/108527147/203794054-0f7dd07c-89fe-4cd4-8bac-30d432dde159.JPG)
the program should randomly generate a number of 4 _different_ digits